### PR TITLE
Update Event “2020/10/2020-10-15-uswds-monthly-call-october-2020”

### PR DESCRIPTION
**Pull request by RhettSkelton**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by RhettSkelton** [comment]
May not build until Ron's author bio is approved, but should be good once Ron's bio is approved.

**Review by saracope — APPROVED** [review]

**Pull request merged**
